lib/takeaway.rb

```diff
@@ -0,0 +1,15 @@
+class Takeaway
+
+  attr_reader :menu
+
+  def initialize
+    @menu = { Cheeseburger: 5,
+              Fries: 3,
+              Milkshake: 2.99  }
+  end
+
+  def menu
```

Comment:
Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. (https://github.com/bbatsov/ruby-style-guide#attr_family)